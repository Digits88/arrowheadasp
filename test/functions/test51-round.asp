<!-- Test Round function.
-->

Round(12) = <%= Round(12) %><br>
Round(12.1) = <%= Round(12.1) %><br>
Round(12.5) = <%= Round(12.5) %><br>
Round(12.7) = <%= Round(12.7) %><br>
Round(13.4) = <%= Round(13.4) %><br>
Round(13.4999) = <%= Round(13.4999) %><br>
Round(13.5) = <%= Round(13.5) %><br>
Round(13.5001) = <%= Round(13.5001) %><br>
Round(13.55) = <%= Round(13.55) %><br>
Round(13.6) = <%= Round(13.6) %><br>
Round(-13.4) = <%= Round(-13.4) %><br>
Round(-13.4999) = <%= Round(-13.4999) %><br>
Round(-13.5) = <%= Round(-13.5) %><br>
Round(-13.5001) = <%= Round(-13.5001) %><br>
Round(-13.55) = <%= Round(-13.55) %><br>
Round(-13.6) = <%= Round(-13.6) %><br>
Round(12.5555, 0) = <%= Round(12.5555, 0) %><br>
Round(12.5555, 1) = <%= Round(12.5555, 1) %><br>
Round(12.5555, 2) = <%= Round(12.5555, 2) %><br>
Round(12.5555, 3) = <%= Round(12.5555, 3) %><br>
