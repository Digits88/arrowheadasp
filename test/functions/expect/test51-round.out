<!-- Test Round function.
-->

Round(12) = 12<br>
Round(12.1) = 12<br>
Round(12.5) = 12<br>
Round(12.7) = 13<br>
Round(13.4) = 13<br>
Round(13.4999) = 13<br>
Round(13.5) = 14<br>
Round(13.5001) = 14<br>
Round(13.55) = 14<br>
Round(13.6) = 14<br>
Round(-13.4) = -13<br>
Round(-13.4999) = -13<br>
Round(-13.5) = -14<br>
Round(-13.5001) = -14<br>
Round(-13.55) = -14<br>
Round(-13.6) = -14<br>
Round(12.5555, 0) = 13<br>
Round(12.5555, 1) = 12.6<br>
Round(12.5555, 2) = 12.56<br>
Round(12.5555, 3) = 12.556<br>
